torch/distributed/_tensor/experimental/attention.py

```diff
@@ -54,6 +54,10 @@ def _merge_sdpa(
     """
     assert len(chunks) == len(logsumexps)
 
+    # LSE may be padded in the sequence dimension such as with memory efficient attention.
+    seq_len = chunks[0].size(2)
```

Comment:
is this a no-op in other attention op?

Reply:
yes exactly -- it's already size seq_len in flash attention. mem efficient attention the length is aligned to 32 len